recipes/veg2hab/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "veg2hab" %}
+{% set version = "1.0.0" %}
+{% set python_min = "3.8" %}
```

Comment:
Conda-forge doesn't support py38. If you really need it we can keep this, but be aware we won't address any issues that may arise from this.

```suggestion
```

Reply:
Thanks for reviewing. It should be fine, I'm about to release a new version of veg2hab, that drops support for the older versions of python.